lcls_tools/common/data_analysis/archiver.py

```diff
@@ -44,6 +56,14 @@ def __hash__(self):
 
     @property
     def timestamp(self) -> datetime:
+        """Gets the datetime object for the ArchiverValue data point at the given time.
+
+        matplotlib can handle datetime objects directly, but data should be cleaned so that HH:MM:SS are also displayed.
+
+        :return self._timestamp: a datetime object corresponding to a data point in the dataset.
+        :rtype self._timestamp: datetime
```

Comment:
Question: is there something parsing this that requires having the type annotation twice? As in, the `-> datetime` is already there

Reply:
No, I think I was going a bit overboard with the docstring annotations. The `-> datetime` is enough.

Reply:
Oh ok! I think we can probably get rid of all the excess annotations then. All of the function signatures should have type annotations (if they don't, feel free to add them)

Reply:
Ok will do.